internal/targets/javascript/javascript.go

```diff
@@ -0,0 +1,134 @@
+package javascript
```

Comment:
I would suggest moving the core logic into `internal/javascript/javascript.go` and just call the logic in the targets.